luigi/contrib/slack_notifications.py

```diff
@@ -0,0 +1,148 @@
+
+'''
+Supports sending slack notifications for different task events.
+
+Usage:
+
+Create an incomming webhook on slack: https://api.slack.com/incoming-webhooks
+
+Use a configuration like the following on luigi.cfg
+.. code-block:: ini
+    [slack]
+    webhook: https://hooks.slack.com/services/...
+    events: ["START", "SUCCESS", "FAILURE"]
+    template: Luigi task {task_id} event {event}: {args} {kwargs}
+
+
+In the python module, setup the slack integration with:
+.. code-block:: python
+    from luigi.contrib.slack_notifications import configure_slack
+    configure_slack()
```

Comment:
Cool! Can this be done from anywhere in your python code?

I'm kinda hinting that the wording "`the python module`" is not super clear to me.